wordpress-mobile/WordPress-iOS
# Giphy: Add attribution image to no results view

As part of our Giphy integration, we need to add a Giphy attribution image to the app. I've added this to the no results view in the Giphy media picker, by adding a new `subtitleImage` view to `NoResultsViewController`. I added a separate initialiser so that a `NoResultsViewController` has _either_ subtitle labels or a subtitle image view.

![simulator screen shot - iphone 8 - 2018-09-05 at 21 11 39](https://user-images.githubusercontent.com/4780/45119040-1f939380-b152-11e8-9a9c-6a65b01778e2.png)

**To test**

* Build and run
* Navigate to the media library for a site and tap the + button in the upper right
* Choose Giphy from the action sheet
* Check you can see the attribution logo in the no results view
